front/src/modules/ui/table/editable-cell/components/EditableCellContainer.tsx

```diff
@@ -38,6 +39,14 @@ export function EditableCellDisplayContainer({
   softFocus,
   onClick,
 }: React.PropsWithChildren<Props>) {
+  const scrollRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
```

Comment:
Nice, simple and efficient. 

If you need useEffects it's better to use them in the soft focus or edit mode children components, thus we're sure that we don't have thousands of useEffect always mounted and potentially triggering and harming perfomance. 

Could you find a way to put the useEffect in a component that is in the hierarchy of soft focus mode ?

Reply:
Sure moved it to CellSoftFocus component -> useEffect only registered in current focus cell